tests/functional/floorplan_reconstruction/get-slot-test.py

```diff
@@ -0,0 +1,70 @@
+# ruff: noqa: INP001
+
+__copyright__ = """
+Copyright (c) 2025 RapidStream Design Automation, Inc. and contributors.
+All rights reserved. The contributor(s) of this file has/have agreed to the
+RapidStream Contributor License Agreement.
+"""
+import json
+import logging
+import os
+
+import pytest
+from python.runfiles import Runfiles
+
+from tapa.common.graph import Graph
+
+GRAPH_PATH = "_main/tests/functional/floorplan_reconstruction/graph/{}-graph.json"
+GOLDEN_PATH = "_main/tests/functional/floorplan_reconstruction/golden/{}-golden.json"
+SLOT_INFO_PATH = "_main/tests/functional/floorplan_reconstruction/slot/{}-slot.json"
+NEW_GOLDEN_FILE = "{}-new-golden.json"
```

Comment:
Do you intend to make these public and importable from other modules? If not, prefer to make them private by prepending an underscore to their names.